climada/engine/unsequa/calc_impact.py

```diff
@@ -24,19 +24,23 @@
 import logging
 import time
 from typing import Union
+import itertools
 
 import pandas as pd
 import numpy as np
+import pathos.multiprocessing as mp
+# use pathos.multiprocess fork of multiprocessing for compatibility
+# wiht notebooks and other environments https://stackoverflow.com/a/65001152/12454103
 
 from climada.engine import ImpactCalc
 from climada.engine.unsequa import Calc, InputVar, UncImpactOutput
+from climada.engine.unsequa.calc_base import _sample_parallel_iterator, _multiprocess_chunksize, _transpose_chunked_data
```

Comment:
Line too long. Black does this:
```suggestion
from climada.engine.unsequa.calc_base import (
    _sample_parallel_iterator,
    _multiprocess_chunksize,
    _transpose_chunked_data,
)
```